Gemfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source 'https://rubygems.org/'
+
+gem 'test-kitchen'
+gem 'kitchen-terraform', '~> 3.3'
```

Comment:
We've been using kitchen-terraform 4.x releases (most recently 4.0.3), and there are some surprising issues that come up when using an older version of kitchen-terraform and using a newer configuration file. Can we bump this to `'~> 4.0.3`?

Reply:
Yep, I think we can bump versions to newer.
PTAL: https://github.com/terraform-google-modules/terraform-google-pubsub/pull/1/commits/697d09fa409981101455ad866571dec5826eaceb